jobs/getjob.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const AWS = require("aws-sdk"); // eslint-disable-line import/no-extraneous-dependencies
+
+const dynamoDb = new AWS.DynamoDB.DocumentClient();
+
+module.exports.getjob = async (event, context, callback) => {
+  const params = {
+    TableName: "jobs",
+    FilterExpression: "jobId = :jobId",
+    ExpressionAttributeValues: {
+      ":jobId": event.pathParameters.jobId
+    }
+  };
+
+  try {
+    let result = await dynamoDb.scan(params).promise();
+    const error = result.$response.error;
+
+    if (error) {
+      console.error(error);
+      const response = {
+        statusCode: error.statusCode,
+        headers: {
+          "Content-Type": "text/plain",
+          "Access-Control-Allow-Origin": "*",
+          "Access-Control-Allow-Credentials": true
+        },
+        body: "Couldn't fetch the jobs."
+      };
+      callback(null, response);
+      return;
+    }
+
+    const response = {
+      statusCode: 200,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Credentials': true,
+      },
+      body: JSON.stringify(result.Items)
```

Comment:
Could you make this return just one item rather than an array? And if there are no items matching the jobId I think it should return a 404 error

Reply:
Thanks for the comments. Just modified based on the two points: 1) return one item instead of an array, 2) return 404 error when not job not found. Please have a another review. Thanks,